tests/sampleTests/sampleTest.GooglePayInCustomerStep.spec.ts

```diff
@@ -1,19 +1,20 @@
 import { test, CustomerStepPreset } from '../';
 
-import { checkoutAfterSignedIn, checkoutBeforeSignedIn, internalOrder, order390, orderPayment, payment } from './GooglePaySampleTestMockingResponses';
+import { checkoutAfterSignedIn, checkoutBeforeSignedIn, googlePay, internalOrder, order390, orderPayment, payment } from './GooglePaySampleTestMockingResponses';
 
 test.describe('Sample Test Group', () => {
     test('Google Pay wallet button is working', async ({assertions, checkout, page}) => {
         // Testing environment setup
         let isSignedIn: boolean = false;
         const responseProps = { status: 200, contentType: 'application/json' };
-        const storeUrl = 'https://my-dev-store-745516528.store.bcdev';
+        const storeUrl = 'https://my-dev-store-117450812.store.bcdev';
```

Comment:
Do we need to change the store url every time?

Reply:
We update tests this time because Buy now cart changed the URL from `**/api/storefront/payments` to
 `/.*\/api\/storefront\/payments\?cartID=.*/`.
Next time, we either think twice about the change, or let the test framework ignore the change.